### PR TITLE
Feature/auth

### PR DESCRIPTION
**Pull request by repejota**
Added auth support

### CODE REVIEW

**Comment by repejota** [comment]
Not merge yet. Work in progress. :pray:

**Comment by josgilmo** [comment]
Passing test after some little changes. Please, don't do the test. Let me keep testing this :)

File: build/logs/clover.xml [hunk 1]
```diff
@@ -1,168 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (adriacidre) [inline build/logs/clover.xml]: Probably the coverage should be ignored.

Comment (josgilmo) [inline build/logs/clover.xml]: It has to be ignore, yes. I'm agree with you @adriacidre

**Pull request merged**